src/main/Timeline.tsx

```diff
@@ -42,7 +42,7 @@ const Timeline: React.FC<{
   setCurrentlyAt: ActionCreatorWithPayload<number, string>,
   setIsPlaying: ActionCreatorWithPayload<boolean, string>,
 }> = ({
-  timelineHeight = 250,
+  timelineHeight = 200,
```

Comment:
Changes look good to me!

One question though, why change the height of the timeline? To compensate for the header? For the resolutions I checked, the addition of the header does cause a scrollbar to appear for resolutions where there previously were none. Maybe I missed one? Or do you just like the look better? :D

Reply:
On a wide-screen monitor, the current code already causes scrollbars to appear because while the overall height is calculated based on the window height, the components in the cutting view seem to have a height calculated so that things just fit in Firefox and Chrome when using a 16:9 aspect ratio.

You can see this when your window is wider than 16:9 if you look at where the menu end on the left side (and you get a scrollbar). Here is a screenshot from editor.opencast.org:

![Screenshot from 2023-04-17 19-54-25](https://user-images.githubusercontent.com/1008395/232571332-17dd4b05-f7c0-4a22-8907-9c0a227141cd.png)

With my patch the common 16:9 resolution now already causes this since there is slightly less vertical space. Adjusting the waveform height takes that into account, restoring the best guess approach from before.

When you complained about the scrollbar, the overall height was also too large. That is fixed properly.

Reply:
I'll admit, I did not check wider than 16:9 resolutions (because I keep thinking this screen is 16:10 while it is actually 16:9...). In that case changing the timeline height seems like a reasonable fix.

In order to avoid this problem in the future, we probably want to introduce a maximum height for the video area at some point. Since the video area always consumes the maximum amount of available width while keeping the video aspect ratios. But that's for another PR.